raphtory/src/db/api/view/internal/materialize.rs

```diff
@@ -141,6 +141,21 @@ impl MaterializedGraph {
         Ok(bincode::serialize_into(&mut writer, &versioned_data)?)
     }
 
+    pub fn save_to_path(&self, path: &Path) -> Result<(), GraphError> {
```

Comment:
we should use the stable encoding if we want to read our graphs back after a structural change